documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+[NOTICE](#NOTICE)
+[Template](#Template)
+[Comments](#Comments)
+
+## DEVNOTES
+
+feature/184 - version 0.2 documentation
+Working on formatting this document like the [spawnPatterns readme.](openscope/src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md).
+Plans:
+    - Remove all "// <explanation>" lines from the template so it can be copy/pasted.
+    - Explain needed properties below the template.
+
+## NOTICE
 
 The airport JSON file must be in `assets/airports`; the filename
 should be `icao.json` where `icao` is the lowercase four-letter ICAO
 airport code, such as `ksfo` or `kmsp`.  If this is a new airport, there
-should also be an entry added to the end of [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js)
+should also be an entry added to the end of [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js).
```

Comment:
Even if it's at the end, it's still alphabetical in-game. We can fix that manually, if needed. But I'll add that note.